tests/common/plugins/sanity_check/recover.py

```diff
@@ -99,23 +100,22 @@ def recover(dut, localhost, fanouthosts, check_results, recover_method):
     else:
         __recover_with_command(dut, method['cmd'], wait_time)
 
+@reset_ansible_local_tmp
+def neighbor_vm_restart_bgpd(node=None):
```

Comment:
I think the code is to recover bgpd, not to **re**start. Restart means stop and then start. Suggest to rename it to `neighbor_vm_recover_bgpd`.